test-result-summary-client/src/Build/TopLevelBuilds.jsx

```diff
@@ -53,7 +51,8 @@ export default class TopLevelBuilds extends Component {
                             return <TopLevelBuildTable url={url} buildName={buildName} type={type} key={j} />
                         });
                     })}
-                </div>);
+                </div>
+                );

```

Comment:
There is no code change in this file. Could you revert the format change?

Reply:
Yes. I am sorry for this, I was testing something in this file and forgot to revert the change.